src/views/partials/board-settings.handlebars

```diff
@@ -0,0 +1,52 @@
+<a href="/board/{{board._id}}">Back</a>
+<div id="boardSettings">
+    <!-- Form for changing the board settings -->
+    <form action="/board/settings/{{board._id}}" method="POST">
+        <input type="hidden" name="_method" value="PATCH"/>
+        <div class="boardNameDiv">
+            <label for="boardName">Name: </label>
+            <input id="boardName" name="boardName" type="text" placeholder="Board Name" value="{{board.boardName}}"/>
+        </div>
+        <div class="colorDiv">
+            <label for="boardColor">Color: </label>
+            <input id="boardColor" name="color"type="text" placeholder="Color" value="{{board.color}}"/>
+        </div>
+        <div class="descriptionDiv">
+            <label for="description">Description: </label>
+            <textarea id="description" name="description" placeholder="Description" value="{{board.description}}"/>
+        </div>
+        <div class="submitButtonDiv">
+            <button type="submit">Submit</button>
+        </div>
+    </form>
+    <!-- form for deleting a board -->
+    {{#if onlyMember}}
+        <form action="/board/remove/{{board._id}}" method="POST">
+            <div class="deleteButtonDiv">
+                <button type="submit">Delete Board</button>
```

Comment:
Mentioned in the list setting html, but these forms with one button don't need to be forms. The requests can get handled in the client side

Reply:
Read the last resolved conversation